app/kumactl/cmd/install/install_transparent_proxy.go

```diff
@@ -207,6 +219,11 @@ runuser -u kuma-dp -- \
 	cmd.Flags().StringArrayVar(&args.ExcludeOutboundTCPPortsForUIDs, "exclude-outbound-tcp-ports-for-uids", []string{}, "tcp outbound ports to exclude for specific UIDs in a format of ports:uids where both ports and uids can be a single value, a list, a range or a combination of all, e.g. 3000-5000:103,104,106-108 would mean exclude ports from 3000 to 5000 for UIDs 103, 104, 106, 107, 108")
 	cmd.Flags().StringArrayVar(&args.ExcludeOutboundUDPPortsForUIDs, "exclude-outbound-udp-ports-for-uids", []string{}, "udp outbound ports to exclude for specific UIDs in a format of ports:uids where both ports and uids can be a single value, a list, a range or a combination of all, e.g. 3000-5000:103,104,106-108 would mean exclude ports from 3000 to 5000 for UIDs 103, 104, 106, 107, 108")
 	cmd.Flags().StringArrayVar(&args.VnetNetworks, "vnet", []string{}, "virtual networks in a format of interfaceNameRegex:CIDR split by ':' where interface name doesn't have to be exact name e.g. docker0:172.17.0.0/16, br+:172.18.0.0/16, iface:::1/64")
+	cmd.Flags().UintVar(&args.Wait, "wait", 0, "specify the amount of time, in seconds, that the application should wait for the xtables exclusive lock before exiting. If the lock is not available within the specified time, the application will exit with an error. Default value 0 means wait forever. To disable this behavior and exit immediately if the xtables lock is not available, use the --dont-wait flag")
+	cmd.Flags().BoolVar(&args.DontWait, "dont-wait", false, "disable the default behavior of waiting for the xtables exclusive lock. If the lock is not available when the application starts, the application will exit immediately with an error")
```

Comment:
should we just have high value of `wait` and not have `dont-wait` flag?

Reply:
Yes. In other words why would you not want to wait?

Reply:
> should we just have high value of wait and not have dont-wait flag?

With value `0` being an equivalent of `wait forever`, without this flag or creating custom type to work with pointers to values it was not possible to differentiate if user provided value `0` (don't wait at all), or if it's a default one

> Yes. In other words why would you not want to wait?

Just wanted to be defensive. I'm not sure if it's possible, but if something else in the system would hold the lock forever, we would just hang.

I'm gonna remove this flag and set a default value of 5s, and change the behavior that by providing `0`, it will mean `don't wait`, not `wait forever`